diff --git a/dist/es/rx/rx.d.ts b/dist/es/rx/rx.d.ts
index 4454ecceee308a6587ff366396c80ffdc238592f..63869142bded5c3dd6fd227d8c15a2e4883f0aee 100644
--- a/dist/es/rx/rx.d.ts
+++ b/dist/es/rx/rx.d.ts
@@ -1,9 +1,7 @@
-import { fromPromise } from 'rxjs/internal-compatibility';
-export { Subject, forkJoin, empty, of, timer, from, defer, Observable, interval, concat as concatObs, merge as mergeObs, race as raceObs, throwError as throwObs, } from 'rxjs';
+export { Subject, forkJoin, empty, of, timer, from, from as fromPromise, defer, Observable, interval, concat as concatObs, merge as mergeObs, race as raceObs, throwError as throwObs, } from 'rxjs';
 export * from 'rxjs/operators';
 export * from './ofType';
 export * from './waitForType';
-export { fromPromise };
 export interface Observer<T> {
     closed?: boolean;
     next: (value: T) => void;
diff --git a/dist/es/rx/rx.js b/dist/es/rx/rx.js
index e4e68bac6671fa141da1e244f42305f8b90cdb44..161cc27131c53b3d4c05b15fcca418db0557da3c 100644
--- a/dist/es/rx/rx.js
+++ b/dist/es/rx/rx.js
@@ -1,7 +1,5 @@
-import { fromPromise } from 'rxjs/internal-compatibility';
-export { Subject, forkJoin, empty, of, timer, from, defer, Observable, interval, concat as concatObs, merge as mergeObs, race as raceObs, throwError as throwObs, } from 'rxjs';
+export { Subject, forkJoin, empty, of, timer, from, from as fromPromise, defer, Observable, interval, concat as concatObs, merge as mergeObs, race as raceObs, throwError as throwObs, } from 'rxjs';
 export * from 'rxjs/operators';
 export * from './ofType';
 export * from './waitForType';
-export { fromPromise };
 //# sourceMappingURL=rx.js.map
\ No newline at end of file
diff --git a/dist/rx/rx.d.ts b/dist/rx/rx.d.ts
index 4454ecceee308a6587ff366396c80ffdc238592f..63869142bded5c3dd6fd227d8c15a2e4883f0aee 100644
--- a/dist/rx/rx.d.ts
+++ b/dist/rx/rx.d.ts
@@ -1,9 +1,7 @@
-import { fromPromise } from 'rxjs/internal-compatibility';
-export { Subject, forkJoin, empty, of, timer, from, defer, Observable, interval, concat as concatObs, merge as mergeObs, race as raceObs, throwError as throwObs, } from 'rxjs';
+export { Subject, forkJoin, empty, of, timer, from, from as fromPromise, defer, Observable, interval, concat as concatObs, merge as mergeObs, race as raceObs, throwError as throwObs, } from 'rxjs';
 export * from 'rxjs/operators';
 export * from './ofType';
 export * from './waitForType';
-export { fromPromise };
 export interface Observer<T> {
     closed?: boolean;
     next: (value: T) => void;
diff --git a/dist/rx/rx.js b/dist/rx/rx.js
index 7c624ffd7fa2be9886c5846cea21e4266df5d18e..0aa935b3a2e0a3b404670d9b9bab7166b8714868 100644
--- a/dist/rx/rx.js
+++ b/dist/rx/rx.js
@@ -1,9 +1,8 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
 var tslib_1 = require("tslib");
-var internal_compatibility_1 = require("rxjs/internal-compatibility");
-exports.fromPromise = internal_compatibility_1.fromPromise;
 var rxjs_1 = require("rxjs");
+exports.fromPromise = rxjs_1.from;
 exports.Subject = rxjs_1.Subject;
 exports.forkJoin = rxjs_1.forkJoin;
 exports.empty = rxjs_1.empty;
